### ISSUE TITLE
Revamp core implementation 

### ISSUE THREAD

**Issue by debloper** [debloper/piosk#53]
I'm considering running browser as [a `systemd` service](https://unix.stackexchange.com/a/676287) (instead of compositor autostart config).

It'll help in 3 ways:

- not rely on the display server, window manager or compositor (ref #52)
- restart the process automatically without needing reboots (ref #41)
- have a well maintained log to check back for crashed (ref #48)

I have very limited bandwidth at the moment, so no promises on `when`.
Feel free to send pull requests if you want the changes made faster.

**Comment by andiohn** [comment]
Yes, absolutely yes. I have memory leaks on one of the websites I'm loading, so it would be good to just 'fail silently and restart the service' instead of having to figure something else out.

**Comment by andiohn** [comment]
I just added a new issue for the pi micro, perhaps that would be a better starting point. I had another idea to put the chrome tabs into a docker container, so we can limit it's ram or priority, but perhaps that's going too far away from 'simple'
